isofit/utils/template_construction.py

```diff
@@ -355,6 +355,9 @@ def __init__(self, lut_config_file: str = None, emulator: bool = False):
         self.aot_550_spacing = 0
         self.aot_550_spacing_min = 0
 
+        self.rte_auto_rebuild = True
```

Comment:
I don't see this key being used anywhere - am I missing it?

Reply:
You're right. Removed with [ff8c787](https://github.com/isofit/isofit/pull/521/commits/ff8c78784d2c1c5b877d704426ac69230380a2b9).